GVFS/GVFS.UnitTests/Common/NuGetUpgraderTests.cs

```diff
@@ -0,0 +1,210 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Mock;
+using GVFS.UnitTests.Mock.Common;
+using Moq;
+using NuGet.Packaging.Core;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GVFS.UnitTests.Common
+{
+    [TestFixture]
+    public class NuGetUpgraderTests

```

Comment:
This class is in `GVFS.UnitTests`, but the tests from #675 are in `GVFS.UnitTests.Windows`. Should the `GitHubUpgrader` be tested in the universal-platform project?

Reply:
Good question - currently, `UpgradeOrchestrator.cs` lives in the `GVFS.Upgrader` project, which is a .NET Framework project. As such, all tests that use `UpgradeOrchestrator` need to live in `GVFS.UnitTests.Windows`.

However, the actual upgraders live in `GVFS.Common`, which exists as netcore project, so they can be tested in `GVFS.UnitTests`.

So - yes, they should be in a universal-platform project (eventually).